src/geodesy.jl

```diff
@@ -11,43 +11,62 @@ import Base.isless
 import Base.isgreater
 import DataStructures.Queue, DataStructures.Deque, DataStructures.enqueue!, DataStructures.dequeue!, DataStructures.dequeue_pair!
 
+_build_connectivity(r::Int, n) = trues(ntuple(_->r, n))
+function _build_connectivity(r::NTuple{N,Int}, n) where N
```

Comment:
Whoa :) i have much stuff to learn on julia 
But in case of reconstruction algorithm we restrain/make sens only for a unit ball circle (or in rare cases segment, if for instance, we made a erosion with some linear SE and want to reconstruct along the segment)

Reply:
a new bug: it should be `r * 2 + 1` here if we interpret `r` as radius :)

Reply:
Ah yes :)
Anyway even if we not use this buildconnectivty helper we could may reuse it elsewhere
Something important to know is that
Erode of size r =2 <==> Erode(erode) os size 1 and its far more efficient to iterate than to use a 'fat' SE 

Basically we define whats is called and homothetic SE, (baseSE,size) where baseSE is something with r=1 and we iterate 1..size
The only counterexemples is if we have a BIG SE which is not a ball, we could/should decompose thirst the SE in smaller composable sub SE